sos/report/plugins/sssd.py

```diff
@@ -23,19 +23,22 @@ class Sssd(Plugin):
 
     def setup(self):
         self.add_copy_spec([
-            # Main config file
+            # main config file
             "/etc/sssd/sssd.conf",
             # SSSD 1.14
             "/etc/sssd/conf.d/*.conf",
-            # Memory cache
-            "/var/lib/sss/mc/*",
-            # Dynamic Kerberos configuration
+            # dynamic Kerberos configuration
             "/var/lib/sss/pubconf/krb5.include.d/*"
         ])
 
         # add individual log files
         self.add_copy_spec(glob("/var/log/sssd/*log*"))
 
+        # add memory cache
+        self.add_copy_spec(glob("/var/lib/sss/mc/passwd"))
+        self.add_copy_spec(glob("/var/lib/sss/mc/group"))
+        self.add_copy_spec(glob("/var/lib/sss/mc/initgroups"))
```

Comment:
Are these files or directories? If they're straight files there's no reason to glob here, and you can just pass the individual file names in a list in a single `add_copy_spec()` call. Each item in a list has the `--log-size` limitation applied to it individually.

Reply:
Thank you for this suggestion, I just updated the code. Please take a look and share you thoughts.